### ISSUE TITLE
args.offset cannot be set using onDataLabelRender callback

### ISSUE THREAD

**Issue by hnnngwdlch** [syncfusion/flutter-widgets#96]
Hi,

according to this [link](https://www.syncfusion.com/feedback/19275) I should now be able to apply offsets to individual data labels.

However, the vertical offset I tried to apply seems to be ignored. Here's my sample code:

```
class TestGraph extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    final List<DisplayedData> columnSeriesData = [
      DisplayedData('Sleeping', 8.7),
      DisplayedData('Personal Care', 0.8),
      DisplayedData('Eating and drinking', 4.5),
      DisplayedData('Housework', 1.9),
      DisplayedData('Other', 1.3),
      DisplayedData('Childcare / care for others', 0.7),
      DisplayedData('Working', 3.8),
      DisplayedData('Education', 5.9),
      DisplayedData('Socialization', 0.7),
      DisplayedData('Watching television', 2.7),
      DisplayedData('Exercise / recreation', 0.3),
      DisplayedData('Leisure', 1.4),
    ];
    return Scaffold(
      backgroundColor: Colors.blue[100],
      body: Padding(
        padding: EdgeInsets.symmetric(
          vertical: 20,
          horizontal: 0,
        ),
        child: Column(
          children: <Widget>[
            Expanded(
              child: SfCartesianChart(
                onDataLabelRender: (DataLabelRenderArgs args) {
                  int index = args.pointIndex;
                  List<CartesianChartPoint<dynamic>> dataPoints =
                      args.dataPoints;
                  if (dataPoints[index].y < 1) {
                    args.textStyle = TextStyle(
                      color: Colors.orange[700],
                    );
                    args.offset = Offset(
                      0,
                      20,
                    );
                  }
                },
                primaryXAxis: CategoryAxis(
                  isVisible: false,
                ),
                series: <ChartSeries>[
                  ColumnSeries<DisplayedData, String>(
                    dataSource: columnSeriesData,
                    xValueMapper: (DisplayedData data, _) => data.x,
                    yValueMapper: (DisplayedData data, _) => data.y,
                    dataLabelSettings: DataLabelSettings(
                      labelAlignment: ChartDataLabelAlignment.bottom,
                      isVisible: true,
                      angle: 90,
                    ),
                  ),
                ],
              ),
            ),
          ],
        ),
      ),
    );
  }
}

class DisplayedData {
  final Object x;
  final num y;

  DisplayedData(
    this.x,
    this.y,
  );
}
```

![image](https://user-images.githubusercontent.com/57846029/103085342-b6194680-45e1-11eb-8e9d-d905e6d0088a.png)

Am I missing something here?

Thanks,
Henning

**Comment by Dharanipalani** [comment]
Hi @hnnngwdlch, 

        Sorry for the inconvenience. We are able to reproduce your issue from our side and we consider this as a bug and it will be fixed in the upcoming patch release which will be rolled out mid of the next week and please be patience till then.

Thanks,
Dharanitharan.P

**Comment by Dharanipalani** [comment]
Hi @hnnngwdlch,

We are glad to inform you that the reported issues have been resolved in our latest version **v18.4.34**. Now, you can able to apply the data label offset values to the individual data labels using the **onDataLabelRender** event. So, kindly upgrade our charts to the latest version from the following link.

Latest version: https://pub.dev/packages/syncfusion_flutter_charts/versions/18.4.34

Please revert us if you need any further assistance.

Thanks,
Dharanitharan. P
